src/server/utils/get-issue-last-acitivity.ts

```diff
@@ -0,0 +1,34 @@
+import {popularityFieldsFragment} from '../../types/gql-types';
+
+export function getIssueLastActivity(
+    assigneeLogin: string, issue: popularityFieldsFragment) {
+  let lastActivity = null;
+
+  // The `issue` variable will include createdAt, but it's untyped due to
+  // Apollo restrictions. Hence the any typing and assignment here.
+  // tslint:disable-next-line:no-any
+  const untypedIssue = issue as any;
+  if (untypedIssue.updatedAt) {
+    lastActivity = new Date(untypedIssue.updatedAt).getTime();
+  }
+
+  if (!issue.comments.nodes) {
+    return lastActivity;
+  }
+
+  if (issue.comments.nodes.length === 0) {
+    return lastActivity;
+  }
+
+  const lastComment = issue.comments.nodes[issue.comments.nodes.length - 1];
```

Comment:
Since we're using `last` in the query, the latest comment is the first one in the list - this should be `nodes[0]`.